tests/python/unittest/test_operator.py

```diff
@@ -8655,6 +8656,17 @@ def test_add_n():
     assert_almost_equal(rslt.asnumpy(), add_n_rslt.asnumpy(), atol=1e-5)
 
 
+def test_get_all_registered_operators():
+    ops = get_all_registered_operators()
+    ok_(isinstance(ops, list))
+    ok_(len(ops) > 0)
+
+
+def test_get_operator_arguments():
+    operator_arguments = get_operator_arguments(mx.operator.get_all_registered_operators()[0])
+    ok_(isinstance(operator_arguments, OperatorArguments))
```

Comment:
Can we validate the correctness using an operator that will most likely not change its arguments? I know it's not 100% safe, but that would make sure that the content is not garbage

Reply:
Ok